public_suffix_list.dat

```diff
@@ -12550,6 +12550,10 @@ curv.dev
 *.ocp.customer-oci.com
 *.ocs.customer-oci.com
 
+// cyber_Folks S.A. : https://cyberfolks.pl
+// Submitted by Bartlomiej Kida <b.kida@cyberfolks.pl>
```

Comment:
Please use a non-personal e-mail!

Reply:
Thank you for noticing the incorrect entry. I changed my email address to a non-personal email address. The current email is the main email regarding the security of the DNS service, I assume it will be the most appropriate one.